packages/ui/components/localize/test/date/parseDate.test.js

```diff
@@ -51,6 +51,57 @@ describe('parseDate()', () => {
     expect(equalsDate(parseDate('12-31-1976'), new Date('1976/12/31'))).to.equal(true);
   });
 
+  it('handles different timezones', () => {
+    /**
+     * @param {Date | undefined} date
+     * @param {String} tzString
+     * @returns {Date | undefined}
+     */
+    function convertTZ(date, tzString) {
+      if (date === undefined) {
+        return undefined;
+      }
+      return new Date(date.toLocaleString('en-US', { timeZone: tzString }));
+    }
+
+    expect(
+      equalsDate(
+        convertTZ(parseDate('01-01-1979'), 'Asia/Manila'),
```

Comment:
If parseDate does `new Date(...)` under the hood, these tests seem to always pass I think, since the timezone info is only added in the test atm.

Reply:
Imho this test still gives a false sense of reliability. Unfortunately it's just not possible to mock the timezone of the browser. Now we have a situation where `equalsDate(parseDate('01-01-1979'), new Date('1979/01/01'))` passes and then we apply the exact same timezone transformation to both dates and test for equality... (which will also pass, but what are we testing then exactly ? 🤔 )

We probably want to add `parseDate(x).getTimezoneOffset === Intl.DateTimeFormat().resolvedOptions().timeZone` as a test instead.
(So we **always use the timezone from the browser, regardless of `localize` config**)